### DISCUSSION TITLE
no 'deepmd' in 02.md

### DISCUSSION THREAD

**Discussion by TheBestOrNothing-gwh** [deepmodeling/dpgen#1144] [Q&A]
when 'dpgen init_bulk' runs out, there is no deepmd in 02.md.
![88ce9d5211fd5652f9226c336cffdb4](https://user-images.githubusercontent.com/52249330/222047565-09eb3212-9c2e-413e-a057-c3b695611d81.png)

**Comment by luowh35** [comment]
unzip zhe f5.....zip, and see what's wrong in the stage 4
